pages/linux/efibootbgr.md

```diff
@@ -0,0 +1,23 @@
+# efibootmgr
+
+> Manipulate the UEFI Boot Manager (the Bootoptions).
+
+- List the current settings:
+
+`efibootmgr `(`-v` for Filepaths)
+
+- Create new Bootoption:
+
+`sudo efibootmgr -c -d {{esp_blockdevice}} -l {{\path\to\file.efi}} -L "{{Label}}"`
+
+- > Specific Example: Adding UEFI Shell v2 as a Boot Options:
```

Comment:
```suggestion
- Add UEFI Shell v2 as a boot option:
```
It isn't necessary to specify "Specific example" nor ">". In addition, "Boot Options" should be in singular and in lowercase.

Reply:
Marking as unresolved, since I can't see any changes to the file itself.